torch/_dynamo/polyfills/__init__.py

```diff
@@ -143,8 +150,13 @@ def dict___eq__(d, other):
     return True
 
 
-def set_symmetric_difference(set1, set2, cls=set):
-    symmetric_difference_set = set()
+def set_symmetric_difference(
+    set1: Iterable[T],
+    set2: Iterable[T],
+    # pyrefly: ignore[bad-function-definition]
```

Comment:
FYI this is https://github.com/facebook/pyrefly/issues/1496. We currently match mypy in not accepting this function signature, but that decision isn't set in stone.